build-system/sauce_connect/OWNERS

```diff
@@ -7,6 +7,7 @@
       owners: [
         { name: 'ampproject/wg-infra' },
         { name: 'rsimha', notify: true }
+        { name: 'danielrozenberg', notify: true }
```

Comment:
Since `build-system` already includes `ampproject/wg-infra` as owner, `estherkim` and I already have the ability to approve changes if necessary. Based on this current file, it looks like maybe the desired solution is removing the team from here, so one of `rsimha` or `danielrozenberg` get auto-assigned to the PR (after which point the notify may or may not be necessary). Just a consideration, approving to unblock

Reply:
It's okay for the team to approve these changes, I just want to get notified of changes here :)